src/styles/index.scss

```diff
@@ -1,3 +1,146 @@
+html {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --blue-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  position: relative;
```

Comment:
Add the padding for the card and remove the horizontal padding everywhere. Also don't need to use position in this task
```suggestion
```